ai/vicuna/playground/deepspeed_config_s2.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
If this folder is only needed for qlora training, should we move it under the 'train_qlora' folder?

Reply:
Updated